src/test/java/app/komunumo/data/service/ParticipationServiceIT.java

```diff
@@ -107,7 +107,7 @@ void joinEventAnonymousUser() {
         assertThat(participations).hasSize(1);
 
         final var user = userService.getUserByEmail(email).orElseThrow();
-        assertThat(user).isNotNull();
+        assertThat(user.email()).isEqualTo(email);
```

Comment:
Again, this change is okay, but it would be better in a separate PR because it does not belong to #288.